sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeUriBuilder.cs

```diff
@@ -111,12 +112,12 @@ public string Snapshot
         /// Gets or sets the Shared Access Signature query parameters, or null
         /// if not present in the <see cref="System.Uri"/>.
         /// </summary>
-        public SasQueryParameters Sas
+        public DataLakeSasQueryParameters Sas
```

Comment:
Should this be using DataLakeSasQueryParameters just like blobs? Otherwise the key property values wouldn't be available from Sas property.

Reply:
Does DataLake support AD?  If it does, I think we want this change.

Reply:
Yes